cognitedata/reveal
# feat(react-components): Added support for storing 3D resources layers state in Url

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/UX-1352

## Description :pencil:
3D resources layering states are stored in Url when layer state are changed

## How has this been tested? :mag:

In Storybook


## Test instructions :information_source:

- `cd react-components && yarn && yarn build && yarn storybook`
- Select `LayersButton` example
- Open the canvas of the story in a new window
- Change the 3D resources data by unselecting any loaded resources in `Layer Button` and confirm that the state changes in Url
- Can copy the Url and load it in new tab.


## Checklist :ballot_box_with_check:
